apps/opik-backend/src/main/java/com/comet/opik/api/filter/ExperimentsComparisonValidKnownField.java

```diff
@@ -3,17 +3,23 @@
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
+import java.util.Arrays;
+import java.util.Optional;
+
 @RequiredArgsConstructor
 @Getter
-public enum ExperimentsComparisonField implements Field {
+public enum ExperimentsComparisonValidKnownField implements Field {
```

Comment:
@andrescrz 

I kept this because otherwise, we would need to create field resolution for known fields. As of now, this is enum-based, and it would practically be a duplication. I believe this is something we can refactor later, but it is not critical. Let me know what you think.